readme.md

```diff
@@ -4,18 +4,35 @@ NVDA (NonVisual Desktop Access) is a free, open source screen reader for Microso
+
+### Documentation
+* [NVDA User Guide](https://www.nvaccess.org/files/nvda/documentation/userGuide.html)
+* [NVDA Developer Guide](https://www.nvaccess.org/files/nvda/documentation/developerGuide.html)
+* [NVDA Add-ons Development Internals](https://github.com/nvdaaddons/DevGuide/wiki)
+* [NVDA ControllerClient manual](https://github.com/nvaccess/nvda/tree/master/extras/controllerClient)
+* Further documentation is included in the Wiki of this repository and in the [Community Wiki](https://github.com/nvaccess/nvda-community/wiki)
+
+### Communication channels
+* [NVDA Users Mailing List](https://nvda.groups.io/g/nvda)
+* [NVDA Developers Mailing List](https://groups.io/g/nvda-devel)
+* [NVDA Add-ons Mailing List](https://nvda-addons.groups.io/g/nvda-addons)
+* [Instant Messaging channel for NVDA Support](https://gitter.im/nvaccess/NVDA)
+* [Other sources including groups and profiles on social media channels, language specific websites and mailing lists etc.](https://github.com/nvaccess/nvda-community/wiki/Connect)
+
+You can also get  direct support from NV Access. See the [NV Access](http://www.nvaccess.org/) website for more details.
+
+## Other Key Project Links
 * [NVDA on GitHub](https://github.com/nvaccess/nvda)
 * [NVDA issues on GitHub](https://github.com/nvaccess/nvda/issues): Bug reports, feature requests, etc.
 * [NVDA development snapshots](https://www.nvaccess.org/files/nvda/snapshots/): Automatically generated builds of the project in its current state of development
 * [NVDA add-ons](https://addons.nvda-project.org/): Get add-ons to enhance NVDA
+* [NVDA Add-ons coordination and support center](https://github.com/nvdaaddons): all about NVDA's addons environment
+* [NVDA Add-ons Template](https://github.com/nvdaaddons/AddonTemplate): A repository for generating the Add-ons template
 * [Translating NVDA](https://github.com/nvaccess/nvda/wiki/Translating): Information about how to translate NVDA into another language
-* [NVDA community wiki](https://github.com/nvaccess/nvda-community/wiki): Articles contributed by the community
 * [NVDA Controller Client](http://www.nvda-project.org/nvdaControllerClient/nvdaControllerClient_20100219.7z) (2010-02-19): NVDA API for external applications to directly speak or braille messages, etc.
-* [NVDA Developer Guide](https://www.nvaccess.org/files/nvda/documentation/developerGuide.html)
 * [Contributing to NVDA](https://github.com/nvaccess/nvda/wiki/Contributing): Guidelines for contributing to the NVDA source code
-* [NVDA development email list](https://nvda-devel.groups.io/) ([Old archives](http://nabble.nvda-project.org/Development-f1.html)): Discussion about NVDA development
```

Comment:
When you've moved informations about NVDA devel above you've forgotten to move link to the old archives along with it